app/scripts/controllers/app-state-controller.ts

```diff
@@ -81,9 +81,6 @@ export type AppStateControllerState = {
   lastInteractedConfirmationInfo?: LastInteractedConfirmationInfo;
   termsOfUseLastAgreed?: number;
   snapsInstallPrivacyWarningShown?: boolean;
-  interactiveReplacementToken?: { url: string; oldRefreshToken: string };
```

Comment:
These properties were only used by MMI (within MMI code fences)

As far as I can tell they were never even initialized on non-MMI builds, so there should be no need for a migration to remove them either.